rewards/classes/RewardsManager.py

```diff
@@ -136,13 +145,33 @@ def calculate_all_sett_rewards(
                 ]
             )
             all_rewards.append(rewards)
-
+            rewards_per_sett[sett]["actual"] = rewards.totals.toDict()
+            rewards_per_sett[sett]["expected"] = expected
+        
         send_code_block_to_discord(
             msg=tabulate(table, headers=["vault", "boosted rewards", "flat rewards"]),
             username="Rewards Bot",
             url=self.discord_url,
         )
+
+        invalid_totals = check_token_totals_in_range(self.chain, rewards_per_sett)
+        if len(invalid_totals):
+            self.report_invalid_totals(invalid_totals)
+            
         return combine_rewards(all_rewards, self.cycle)
+    
+    def report_invalid_totals(self, invalid_totals: List[List[str]]) -> None:
```

Comment:
Maybe not for now, but RewardsManager is getting pretty large atm. We can have this method as a util function or something

Reply:
agreed will write ticket